lib/train/transports/winrm.rb

```diff
@@ -100,9 +103,9 @@ def validate_options(opts)
     def connection_options(opts)
       {
         logger:                   logger,
-        transport:                :negotiate,
-        disable_sspi:             false,
-        basic_auth_only:          false,
+        transport:                opts[:transport].intern,
```

Comment:
It's equivalent, but more readable, to say `.to_sym` here.

Reply:
Actually, come to think of it, we might want to add some validation here; I'm not sure what valid values are for the transport flag.

Reply:
resolved.